packages/internet-header/src/components/post-search/post-search.tsx

```diff
@@ -377,6 +396,17 @@ export class PostSearch implements HasDropdown, IsFocusable {
                         onKeyDown={e => this.handleKeyDown(e)}
                       />
                       <label htmlFor="searchBox">{translations.flyoutSearchBoxFloatingLabel}</label>
+                      <button
+                        onClick={() => this.handleClearSearchbar()}
+                        class="visually-hidden clear-search-button"
+                        type="reset"
+                        aria-label="Clear input"
+                        id="clearButton"
+                        ref={el => (this.clearButton = el)}
+                        title="Clear input"
```

Comment:
A title is a good idea, but the header currently shows no titles on other icon buttons.

For consistency I suggest you remove the title from this PR and perhaps create a ticket to add titles to all buttons at once.